packages/inference/src/HfInference.ts

```diff
@@ -684,6 +684,10 @@ export class HfInference {
 		if (this.apiKey) {
 			headers["Authorization"] = `Bearer ${this.apiKey}`;
 		}
+    
+    if (!options?.binary) {
```

Comment:
Last nit: the indent should be tabs here

Reply:
Oops! I added the edit right on the GitHub merge conflict editor, so Prettier didn't fix it. I have pushed a formatting fix in the latest commit. Thank you!